src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -249,6 +285,35 @@ class PackageDetailTab extends mixin(StoreMixin) {
     }
   }
 
+  handlePackageVersionChange(packageOpt) {
```

Comment:
This seems like the kind of thing we'd have a util for? It's just a function that sets a value from the query string.

Reply:
I argue against a util because promotes reusability and we want this being used only for this component.

Reply:
I'm more just surprised that there wouldn't be somewhere else in the codebase we do this exact same type of thing.